cron/auth02mozdef.py

```diff
@@ -308,45 +358,24 @@ def process_msg(mozmsg, msg):
         pass
 
     try:
-        details['auth0_client_id'] = msg.client_id
+        details['clientid'] = msg.client_id
     except KeyError:
         pass
 
-    try:
-        details['username'] = msg.details.request.auth.user.name
-        details['action'] = msg.details.response.body.name
-    except KeyError:
-        try:
-            details['error'] = 'true'
-            details['errormsg'] = msg.details.error.message
-        except KeyError:
-            pass
-        except AttributeError:
-            pass
-        try:
-            details['username'] = msg.user_name
-        except KeyError:
-            pass
-
     # Differenciate auto login (session cookie check validated) from logged in and had password verified
 
     try:
         for i in msg.details.prompt:
             # Session cookie check
             if i.get('name') == 'authenticate':
-                details['auth_type'] = 'Login succeeded due to a valid session cookie being supplied'
```

Comment:
I could be the minority here, but I actually prefer to have the key names contain the '_' to separate words like this, it seems more legible for me.

Reply:
Not impossible, but if we wanted to be consistent we would have to change the standard so things like sourceipv4address become source_ip_v4_address, destination_port, source_port, etc. Probably a discussion for another issue/pr.